### DISCUSSION TITLE
Run examples/action_primitives/wip_solve_behavior_task.py failed

### DISCUSSION THREAD

**Discussion by ClorisUcas** [StanfordVL/BEHAVIOR-1K#1018] [Q&A]
Thanks for your great contribution in robotic field.
When I run examples/action_primitives/wip_solve_behavior_task.py, "Fetch" robot cannot grasp the can. I record the video, and it seems like the arm of  robot cannot reach the can. Is there any good suggestion for me to solve this problem？
[fetch_wip_solve_behavior.zip](https://github.com/user-attachments/files/17528621/fetch_wip_solve_behavior.zip)
Looking for your helpful reply :)

**Comment by hang-yin** [comment]
Hi @ClorisUcas , please refer to this post: https://github.com/StanfordVL/OmniGibson/discussions/987
